### PR TITLE
Import correction of tkinter

### PR DESCRIPTION
**Pull request by wi6n3l**
Import correction of tkinter and ttk

### CODE REVIEW

**Comment by wi6n3l** [comment]
Import correction of tkinter

**Comment by 5hubh4m** [comment]
The new `import`s work on `python 3` not `python 2`.

**Comment by wi6n3l** [comment]
But tour program is written for python 3 :/

File: chatup.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env python2.7
```
Comment (5hubh4m) [inline chatup.py:1]: No, it's for `python 2`.

**Comment by wi6n3l** [comment]
Do u mine if i make a python 3 version of it?

**Comment by 5hubh4m** [comment]
Not really. Create another branch for it.

**Pull request closed without merging**